nav2_bt_navigator/behavior_trees/odometry_calibration.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+  This Behavior Tree drives in a square
+-->
+
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <Sequence name="Drive in a square">
+      <DriveOnHeading dist_to_travel="0.2" speed="0.02" time_allowance="12"/>
+      <Spin spin_dist="1.57"/>
```

Comment:
Add a few more sig figs than that, that much could actually contribute to perceived issues if repeated 100 times. At least 5 decimals would be nice. I doubt it'll execute that accurately, but at least good to know we set it to be enough where the BT wouldn't be the cause of any issues